aws_access/test_aws_access.py

```diff
@@ -87,7 +87,7 @@ def test_grant_aws_access(
         "Access.access_modules.aws_access.helpers.get_aws_client",
         return_value=boto3_client,
     )
-    mocker.patch("bootprocess.general.emailSES", return_value="")
+    mocker.patch("Access.base_email_access.access.BaseEmailAccess.email_via_smtp", return_value="")
```

Comment:
```suggestion
    mocker.patch("Access.access_modules.aws_access.access.AWSAccess.email_via_smtp", return_value="")
```
does this make more sense? lets mock the same method that is being called - and not the parent
reason: we would like to minimize references to files in enigma repo (access.base_email_access is being inherited once) - this is being referenced in base module once, lets not reference them in tests if not needed

Reply:
makes sense. Done.